Not possible to include js files referenced by relative path in test files

samyhrer opened the pull request:
Returning absolute paths broke the ability to include js files
referenced by relative path in test files

samyhrer committed:
getPaths now returns relative paths of test files

Returning absolute paths broke the ability to include js files
referenced by relative path in test files

davglass commented:
I'll take a look at this, this was added to support Windows so I need to make sure this doesn't break that ;)

davglass commented:
Do you have a test case for this? I've tested it locally without modification and it seems to work as expected. I'd like to see what I'm missing with this.

samyhrer commented:
I'm having a setup with this directory structure and these files:

/src/test.js
/test/rel.html

Running 
grover test/**/*.html
throws "javascript error script timeout" after a while.

Contents of rel.html:

``` html
<!DOCTYPE HTML>
<html>
<head>
    <meta http-equiv="content-type" content="text/html; charset=utf-8">
    <title>Basic Usage Example</title>
    <script src="http://yui.yahooapis.com/3.6.0/build/yui/yui-min.js"></script>
</head>
<body>
    <script src="../src/test.js"></script>
</body>
</html>
```

The use case is to be able to reference both test-scripts as well as other js-dependencies by relative path

samyhrer committed:
Allow to directly reference multiple yui test files on the windows platform

On the windows platform it was not possible to start test this way:
grover <path to test file> <path to test file>

samyhrer committed:
Allow to reference yui test files by advanced glob patterns on *nix

Glob patterns such as /**/*.html was not supported on *nix-platforms.
This is now introduced by the -g <pattern> argument.

SebastianStiehl commented:
That works for me too. Thx samyrer.
